docs/diagrams/src/PostFreesurferPipelineLongPrep.GFC.puml

```diff
@@ -0,0 +1,54 @@
+@startuml
```

Comment:
Are all of these `.puml` files supposed to have figures associated with them?  If not, do they belong in the repo itself?

Reply:
Yes. PUML sources/Figures are stored under docs/src and docs/out, respectively.
I updated all of the PUML and figures to reflect the up to date architecture.